code/_globalvars/lists/mobs.dm

```diff
@@ -55,3 +56,17 @@ GLOBAL_LIST_EMPTY(mob_config_movespeed_type_lookup)
 	for(var/i in GLOB.mob_list)
 		var/mob/M = i
 		M.update_config_movespeed()
+
+/proc/init_emote_list()
+	. = list()
+	for(var/path in subtypesof(/datum/emote))
+		var/datum/emote/E = new path()
+		if(!.[E.key])
+			.[E.key] = list(E)
+		else
+			.[E.key] |= E

```

Comment:
You shouldn't need |= here
```suggestion
			.[E.key] += E
```

Reply:
If you want to codegolf a bit you could skip the conditional here and just do
`.[E.key] += list(E)`